CoreScriptsRoot/CoreScripts/GamepadMenu.lua

```diff
@@ -667,9 +669,16 @@ local function setupGamepadControls()
 			setButtonEnabled(button, enabled)
 		end
 	end
+	
+	local function enableRadialMenu()
+		ContextActionService:BindCoreAction(toggleMenuActionName, doGamepadMenuButton, false, Enum.KeyCode.ButtonStart)
+	end
+	
+	local loadedConnection = game.Loaded:connect(enableRadialMenu)
 	StarterGui.CoreGuiChangedSignal:connect(setRadialButtonEnabled)
-
-	ContextActionService:BindCoreAction(toggleMenuActionName, doGamepadMenuButton, false, Enum.KeyCode.ButtonStart)
+	
+	game.Close:wait()
+	loadedConnection:disconnect()
```

Comment:
Oh right; that should only run once, so there's no point in leaving the event open for the duration of the game.